wolfssl/wolfio.h

```diff
@@ -125,7 +125,7 @@
         #elif defined(EBSNET)
             #include "rtipapi.h"  /* errno */
             #include "socket.h"
-        #elif !defined(DEVKITPRO) && !defined(WOLFSSL_PICOTCP) && !defined(WOLFSSL_CONTIKI)
+        #elif !defined(DEVKITPRO) && !defined(WOLFSSL_PICOTCP) && !defined(WOLFSSL_CONTIKI) && !defined(WOLFSSL_WICED)
```

Comment:
Let's try and keep the max line length to 80 and for macro's use backslash between lines.